tests/ga/test_extension.py

```diff
@@ -1620,13 +1620,41 @@ def test_ext_path_and_version_env_variables_set_for_ever_operation(self, *args):
 
             # Extension Path and Version should be set for all launch_command calls
             for args, kwargs in patch_start_cmd.call_args_list:
-                self.assertIn(EXTENSION_PATH, kwargs['env'])
-                self.assertIn('OSTCExtensions.ExampleHandlerLinux-1.0.0', kwargs['env'][EXTENSION_PATH])
-                self.assertIn(EXTENSION_VERSION, kwargs['env'])
-                self.assertEqual("1.0.0", kwargs['env'][EXTENSION_VERSION])
+                self.assertIn(ExtCommandEnvVariable.ExtensionPath, kwargs['env'])
+                self.assertIn('OSTCExtensions.ExampleHandlerLinux-1.0.0', kwargs['env'][ExtCommandEnvVariable.ExtensionPath])
+                self.assertIn(ExtCommandEnvVariable.ExtensionVersion, kwargs['env'])
+                self.assertEqual("1.0.0", kwargs['env'][ExtCommandEnvVariable.ExtensionVersion])
 
         self._assert_handler_status(protocol.report_vm_status, "Ready", expected_ext_count=1, version="1.0.0")
 
+    def test_ext_sequence_no_should_be_set_for_every_command_call(self, *args):
+        test_data = WireProtocolData(DATA_FILE_MULTIPLE_EXT)
+        exthandlers_handler, protocol = self._create_mock(test_data, *args)
+
+        with patch.object(CGroupConfigurator.get_instance(), "start_extension_command") as patch_start_cmd:
```

Comment:
could we use an actual extension command to check the environment? or mock at the level of popen? this would make the test a little less dependent on implementation details

Reply:
Yes makes sense, I'll add a test case for that too and leave this test as well (extra test cases never hurt)

Reply:
ok, but then do mock at the level of Popen... I'm not convinced about taking a dependency on the fact we go thru the cgroup configurator to execute extensions. That code has already changed a couple of times.

Reply:
Done